### PR TITLE
PXP-6485 Requestor deployment

### PR DESCRIPTION
**Pull request by paulineribeyre**
Jira Ticket: [PXP-6485](https://ctds-planx.atlassian.net/browse/PXP-6485)

deployment files for [Requestor service](https://github.com/uc-cdis/requestor)

### New Features
- Requestor deployment

### CODE REVIEW

File: kube/services/requestor/requestor-deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      maxSurge: 1
+      maxUnavailable: 0
+  template:
+    metadata:
+      labels:
+        app: requestor
+        release: production
+        public: "yes"
+        # for network policy
+        netnolimit: "yes" # TODO do i need this?
```
Comment (paulineribeyre) [inline kube/services/requestor/requestor-deploy.yaml:24]: @diw2

File: kube/services/requestor/requestor-service.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: requestor-service
+  annotations:
+    getambassador.io/config: | # TODO do i need this?
```
Comment (paulineribeyre) [inline kube/services/requestor/requestor-service.yaml:6]: @diw2

File: kube/services/requestor/requestor-deploy.yaml (hunk 1 above)
Comment (diw2) [reply kube/services/requestor/requestor-deploy.yaml:24]: yes you need this

File: kube/services/requestor/requestor-deploy.yaml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+            readOnly: true
+            mountPath: "/src/requestor-config.yaml"
+            subPath: "requestor-config.yaml"
+        resources:
+          requests:
+            cpu: 0.4
+            memory: 512Mi
+          limits:
+            cpu: 0.8
+            memory: 512Mi
```
Comment (frickjack) [inline kube/services/requestor/requestor-deploy.yaml:81]: requests and limits should not be set to the same value.   Request is what you require to get started, limits is if you go over that you die.  Set limits to 1024 if requests is 512

File: kube/services/requestor/requestor-service.yaml (hunk 2 above)
Comment (frickjack) [reply kube/services/requestor/requestor-service.yaml:6]: no - you can remove this - we started porting reverse proxy to ambassador, but never finished

File: kube/services/requestor/requestor-deploy.yaml (hunk 1 above)
Comment (frickjack) [reply kube/services/requestor/requestor-deploy.yaml:24]: "netnolimit" gives unrestricted network access - only do that for services that need that

Comment (paulineribeyre) [reply kube/services/requestor/requestor-deploy.yaml]: turns out i need do it - the service is unable to fetch keys to decode tokens without it

**Pull request merged**